### PR TITLE
Nandiini

### PR DESCRIPTION
**Pull request by Nandiniverma2024**

### CODE REVIEW

File: src/atoms/atoms-nandini/buttons/bold/yellow/index.astro
Comment (tanishdotdev) [inline src/atoms/atoms-nandini/buttons/bold/yellow/index.astro:1]: You don't have to mention font-weight in buttons.
Update the path: src/atoms/atoms-nandini/buttons/yellow/index.astro

File: src/atoms/atoms-nandini/buttons/medium/gradient/index.astro
Comment (tanishdotdev) [inline src/atoms/atoms-nandini/buttons/medium/gradient/index.astro:1]: You don't have to mention font-weight in buttons.
Rename the button from gradient to indigo-yellow-gradient.

src/atoms/atoms-nandini/buttons/indigo-yellow-gradient/index.astro

File: src/atoms/atoms-nandini/headings/one/4xl/bold/index.astro
Comment (tanishdotdev) [inline src/atoms/atoms-nandini/headings/one/4xl/bold/index.astro:1]: Update the path: src/atoms/atoms-nandini/headings/one/4xl/bold/white/index.astro

/text-size/font-weight/font-color( except black )/

File: src/atoms/atoms-nandini/headings/two/4xl/bold/index.astro
Comment (tanishdotdev) [inline src/atoms/atoms-nandini/headings/two/4xl/bold/index.astro:1]: Update the path: src/atoms/atoms-nandini/headings/one/4xl/bold/center/index.astro

File: src/components/home/about/index.astro
Comment (tanishdotdev) [inline src/components/home/about/index.astro:1]: 1. When naming button, Call it Button followed by variant. For eg: ButtonBlue, ButtonPrimary etc.
2. Follow the chronology of imports
Layouts
Components
Atoms
Props

File: src/components/home/hero/index.astro
Comment (tanishdotdev) [inline src/components/home/hero/index.astro:1]: Create atom for video.

**Review by tanishdotdev — CHANGES_REQUESTED** [review]

File: src/atoms/atoms-nandini/buttons/bold/yellow/index.astro
Comment (Nandiniverma2024) [reply src/atoms/atoms-nandini/buttons/bold/yellow/index.astro:1]: okay

**Comment by Nandiniverma2024** [comment]
Done, In commit messages I wrote (by mistake) home in place of about section

**Pull request closed without merging**